Islandora/islandora
# Delete content node along with its media and file

**GitHub Issue**: [Deleting content node does not delete media/files associated with node](https://github.com/Islandora/documentation/issues/2043)

# What does this Pull Request do?

This PR adds a checkbox for deleting media in the delete form of islandora objects and lists all the media associated with the object in the form of list 

# What's new?

* Does this change add any new dependencies? no
* Does this change require any other modifications to be made to the repository
 (i.e. Regeneration activity, etc.)? no
* Could this change impact execution of existing code? no

# How should this be tested?

Create some content of the Islandora repository item type and add media to that content using media of field. click the content you created from the content page. In the available tabs for the content, click 'Delete' to see the changes.

the feature can be turned on and off with a checkbox in islandora settings page.

# Documentation Status

* Does this change existing behaviour that's currently documented? no
* Does this change require new pages or sections of documentation? probably, yes
* Who does this need to be documented for?
* Associated documentation pull request(s): ___  or documentation issue ___

# Additional Notes:
Any additional information that you think would be helpful when reviewing this
 PR.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora/committers
